shelley/chain-and-ledger/executable-spec/test/Test/Shelley/Spec/Ledger/Rules/TestNewEpoch.hs

```diff
@@ -99,5 +100,5 @@ preservationOfAda tr =
                 }
           }
         ) =
-        reserves + treasury + sum_ rewards + balance utxo + fees + deposits_
-          == reserves' + treasury' + sum_ rewards' + balance utxo' + fees' + deposits'
+        reserves + treasury + sum_ rewards + (getAdaAmount $ balance utxo) + fees + deposits_
```

Comment:
Surely this should be in `Value`? Otherwise this property no longer checks preservation of other assets.

Reply:
Same applies for basically all the others.

Reply:
this is tricky because this doesn't look at the Tx itself, only compares ledger states - so there is no way to inspect the forge. I made this a strictly Ada comparison